### ISSUE TITLE
Improve ReadTheDocs

### ISSUE THREAD

**Issue by naveedishaque** [HiDiHlabs/ssam-lite#18]
TOC
 - no link to landing page (first link is TLDR)

TLDR
 - start with "SSAM-lite is an easy to use a light weight web browser application to identify cell types in single-molecule spatial transcriptomic data such as MERFISH, SEQfish, ISS/CARTANA, osmFISH, etc."
 - add images of the different parts of the analysis (osmFISH dataset)

Installation
 - maybe just say it is supported by "all modern web browsers that support Javascript ECMAScript 5"
 - to visually clarify the difference between solo and server, add the cartoon graphics of SSAM-lite and SSAM-lite-server from the fig1 in the google doc
 - instructions are all linux CLI based, but it should be Windows/MAC GUI friendly 
 - SSAM-lite-server should be written for Linux CLI

Analysis
- Parameters should be visualised - e.g. the effect of parameter changes should be demonstrated on the osmFISH dataset
- Add screen shot for the saving of plotly figures

Solo or server
  - to visually clarify the difference between solo and server, add the cartoon graphics of SSAM-lite and SSAM-lite-server from the fig1 in the google doc

**Comment by niklasmueboe** [comment]
TOC
- not sure I agree but I added it

TLDR
- Should we really add Images for the "TLDR", I mean we have those anyway in the "Usage" chapter, and after all it is an tldr and should be kept as short as possible right?

Installation
- I added a link directly to the zip but there is not more I can do for "Windows/Mac GUI instructions". It already specifies: download -> extract and after all that is everything there is
